urdf/orbbec_astra.urdf.xacro

```diff
@@ -93,57 +99,57 @@ limitations under the License.
-      <child link="${device_namespace}_color_frame" />
+      <parent link="${component_name_underscore}link" />
+      <child link="${component_name_underscore}color_frame" />
     </joint>
 
-    <link name="${device_namespace}_depth_frame" />
+    <link name="${component_name_underscore}depth_frame" />
 
-    <joint name="${parent_link.rstrip('_link')}_to_${device_namespace}_depth_joint" type="fixed">
+    <joint name="${component_name_underscore}to_${component_name_underscore}depth_joint" type="fixed">
       <origin xyz="0.0 0.0 0.03325" rpy="0.0 0.0 0.0" />
-      <parent link="${device_namespace}_link" />
-      <child link="${device_namespace}_depth_frame" />
+      <parent link="${component_name_underscore}link" />
+      <child link="${component_name_underscore}depth_frame" />
     </joint>
 
-    <link name="${device_namespace}_color_frame" />
+    <link name="${component_name_underscore}color_frame" />
 
-    <joint name="${device_namespace}_color_to_${device_namespace}_color_optical_joint" type="fixed">
+    <joint name="${component_name_underscore}color_to_${component_name_underscore}color_optical_joint" type="fixed">
       <origin xyz="0.0 0.0 0.0" rpy="${-pi/2} 0.0 ${-pi/2}" />
-      <parent link="${device_namespace}_color_frame" />
-      <child link="${device_namespace}_color_optical_frame" />
+      <parent link="${component_name_underscore}color_frame" />
+      <child link="${component_name_underscore}color_optical_frame" />
     </joint>
 
-    <link name="${device_namespace}_depth_optical_frame" />
+    <link name="${component_name_underscore}depth_optical_frame" />
 
-    <joint name="${device_namespace}_depth_to_${device_namespace}_depth_optical_joint" type="fixed">
+    <joint name="${component_name_underscore}depth_to_${component_name_underscore}depth_optical_joint" type="fixed">
       <origin xyz="0.0 0.0 0.0" rpy="${-pi/2} 0.0 ${-pi/2}" />
-      <parent link="${device_namespace}_depth_frame" />
-      <child link="${device_namespace}_depth_optical_frame" />
+      <parent link="${component_name_underscore}depth_frame" />
+      <child link="${component_name_underscore}depth_optical_frame" />
     </joint>
 
-    <link name="${device_namespace}_color_optical_frame" />
+    <link name="${component_name_underscore}color_optical_frame" />
 
     <!-- GAZEBO -->
     <!-- https://github.com/orbbec/OrbbecSDK_ROS2#all-available-topics -->
     <xacro:include filename="$(find husarion_components_description)/urdf/gz_sensor.urdf.xacro" ns="gz_sensor" />
 
     <xacro:gz_sensor.camera
-      reference="${device_namespace}_link"
-      name="${ns}${device_namespace}_orbbec_astra_color"
-      topic="${ns}${device_namespace}/color/image_raw"
-      frame_id="${ns}${device_namespace}_color_optical_frame"
+      reference="${component_name_underscore}link"
+      name="${robot_namespace_slash}${component_name_underscore}orbbec_astra_color"
+      topic="${robot_namespace_slash}${component_name_slash}color/image_raw"
```

Comment:
We aim to have the default configuration the same as for the real sensor when possible. Astra afaik default topic is `/camera/color/image_raw` so the default `component_name` should also be `camera`. This also applies to other sensors.